### PR TITLE
Added Sunspot data import and plot example

### PR DESCRIPTION
**Pull request by abotiamnot**
Added a small, documented example on how to use Sunspot and plot its data.

### CODE REVIEW

**Comment by abotiamnot** [comment]
@dstansby Can you tell me why is CircleCI failing?

**Comment by dstansby** [comment]
The example needs a docstring at the top, which includes an underlined title and a short 1 sentence description of the example - see `plot_importing_data.py` for an example of a docstring at the top.

File: examples/plot_sunspot.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+"""
+Importing Sunspot Data
+==============
```
Comment (dstansby) [inline examples/plot_sunspot.py]: The underline needs to be the same length as the title.

File: examples/plot_sunspot.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+"""
+Importing Sunspot Data
+==============
+
+Since a Sunspot Data Import functionality has been added,
+This example aims to demonstrate how can the data be imported and plotted.
```
Comment (dstansby) [inline examples/plot_sunspot.py]: Just this line (and not the line above) is fine for the description.

File: examples/plot_sunspot.py (hunk 1 above)
Comment (abotiamnot) [reply examples/plot_sunspot.py]: Got it! Making changes!

**Comment by abotiamnot** [comment]
I'll add CircleCI to my Github Fork as well, so I can verify my commits on my fork before making a PR.

**Pull request merged**